### PR TITLE
Matching boards for the V0 Mod

### PR DESCRIPTION
**Pull request by JNP-1**
This pull request includes all boards that I used in my V0 Mod. All necessary files are available to order the boards yourself, parts lists and mechanical fastening

### CODE REVIEW

**Comment by timmit99** [comment]
@JNP-1 would you be willing to add a description to the merge request? with 80+ files it would be helpful to have an overview of what we are looking at for a merger =)

Thanks!

**Comment by JNP-1** [comment]
of course

File: V0_PCB_Mod/PCBs/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The replica is done at your own risk.
+
+The circuit boards are for private use only
+
+All parts can be obtained from Mouser or Digikey.
+
+![Image 1](Fan_PCB/Images/1.jpg)
+![Image 2](Hotend_PCB/Images/1.jpg)
+![Image 3](Supply_PCB/images/1.jpg)
```
Comment (john--) [inline V0_PCB_Mod/PCBs/readme.md:15]: I think this should be Images with an. upper-case I. The image fails to load in the preview.

Comment (JNP-1) [reply V0_PCB_Mod/PCBs/readme.md]: Thanks for the hint. I will correct the mistake.

**Review by john-- — APPROVED** [review]

**Pull request merged**